package.json

```diff
@@ -1,14 +1,20 @@
 {
   "name": "moment-relative",
-  "version": "1.1.0",
+  "version": "2.0.0",
   "description": "Support for graphite-style relative times in Moment",
   "main": "index.js",
   "repository": {
     "type": "git",
     "url": "git://github.com/bbc/moment-relative.git"
   },
+  "engines": {
+    "node": ">4.0.0"
+  },
   "scripts": {
-    "test": "mocha"
+    "test": "mocha",
+    "lint": "eslint .",
+    "posttest": "npm run lint",
+    "coverage": "istanbul cover _mocha -- -R dot"
```

Comment:
While we're at it, it might be nice to change this to use `nyc`. Another PR, perhaps.

Reply:
Yes, we can address that separately